README.md

```diff
@@ -84,7 +84,7 @@ function* treeWalker(refresh) {
     const isOpened = yield refresh
       ? {
           id,
-          isLeaf: children.length === 0,
+          isLeaf: children?.length === 0,
```

Comment:
```suggestion
          isLeaf: !children || children.length === 0,
```
If `children` does not exist, the result will be `undefined === 0` which is `false`. It means that the node without children (the leaf) will not be marked as a leaf which could lead to errors.